remote/advancedcontainers/docker-options.md

```diff
@@ -0,0 +1,112 @@
+---
+Order: 16
+Area: advancedcontainers
+TOCTitle: Docker Options
+PageTitle: Remote - Containers Docker options
+ContentId:
+MetaDescription: Remote - Containers Docker options
+DateApproved: 2/4/2022
+---
+
+# Alternate ways to install Docker
+
+You may harness Docker in the Remote - Containers extension in a few ways:
+* Docker installed locally
+* Docker installed on another machine or remote environment
+* Other Docker compliant CLIs, installed locally or in a remote environment
+     * Remote - Containers interacts with CLIs; it makes no assumptions about how a container engine works and does not interact with container engines or daemon directly.
+     * Please note that other Docker compliant CLIs are not officially supported.
+
+Continue reading to learn alternate ways you may install and leverage Docker or a Docker compliant CLI.
+
+## Podman
+
+[Podman](https://podman.io/) 1.9+ is mostly compatible with Docker's CLI commands and therefore generally does work if you update the `Docker Path` setting (which may be accessed through **Remote > Containers: Docker Path**) to `podman` on Linux.
```

Comment:
There is some discussion around using the following to work around issues with rootless Podman and us using HOME in https://github.com/microsoft/vscode-remote-release/issues/4443:
```json
"runArgs": [
  "--userns=keep-id"
],
"containerEnv": {
  "HOME": "/home/node"
}
```
Something we could improve on.

Reply:
Thanks for the context. I added this snippet and some context from the issue. If we change the behavior, we could update the doc, but seems like a popular enough issue at this point to document the workaround?